metacluster/include/metacluster/ListTenants.actor.h

```diff
@@ -101,10 +121,15 @@ Future<std::vector<std::pair<TenantName, MetaclusterTenantMapEntry>>> listTenant
     TenantNameRef begin,
     TenantNameRef end,
     int limit,
-    TenantGroupName tenantGroup) {
-	std::vector<std::pair<TenantName, int64_t>> matchingTenants = wait(listTenantsTransaction(tr, begin, end, limit));
+    TenantGroupName tenantGroup = ""_sr) {
```

Comment:
You may need to use `Optional<TenantGroupName>` if you don't want it to be required. I don't know that it's strictly disallowed to have an empty tenant group name